keyvalue/clients/pom.xml

```diff
@@ -28,6 +28,12 @@
       <artifactId>tachyon-keyvalue-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <!-- Provides javax.annotation -->
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
```

Comment:
I've removed, they are brought in in the original repo where jsr305 is not added to tachyon-common.